torch_xla/csrc/runtime/pjrt_computation_client.cpp

```diff
@@ -554,13 +554,18 @@ std::vector<ComputationClient::ComputationPtr> PjRtComputationClient::Compile(
 
   for (auto& instance : instances) {
     xla::CompileOptions compile_options;
+    for (auto& option : custom_compile_options_) {
```

Comment:
Make the reference const and use structured binding:

`const auto& [name, value] : ...`